dwave/system/testing.py

```diff
@@ -366,7 +397,7 @@ def sample(self, bqm, **kwargs):
                           if pair[1]!=3],dtype=float),
                 [pair[0] for pair in initial_state if pair[1]!=3])
 
-        ss = SteepestDescentSampler().sample(bqm, **substitute_kwargs)
+        ss = self.substitute_sampler.sample(bqm, **substitute_kwargs)
```

Comment:
We need to document that for the first sample the substitution is ignored unless self.exact_solver_cutoff=0; i.e. for small problems the first sample is always a ground state. See comment in tests.